felix/bpf-gpl/fib.h

```diff
@@ -249,4 +249,29 @@ static CALI_BPF_INLINE int forward_or_drop(struct cali_tc_ctx *ctx)
 	return TC_ACT_SHOT;
 }
 
+static CALI_BPF_INLINE int hep_rpf_check(struct cali_tc_ctx *ctx) {
```

Comment:
What if we use `bool` if the answer is yes/no? I wonder if it would make sense to move both `wep/hep_rpf_check` to an `rpf.h` then conntrack would not need to include `fib.h` which is a bit weird.

```suggestion
static CALI_BPF_INLINE bool hep_rpf_check(struct cali_tc_ctx *ctx) {
```